dds_web/api/user.py

```diff
@@ -5,44 +5,40 @@
 ###############################################################################
 
 # Standard library
-import datetime
+from datetime import datetime, timedelta
```

Comment:
We (dev group) decided on only importing modules, not functions and classes, if not absolutely necessary

Reply:
Done!